src/components/SingleArticle.jsx

```diff
@@ -22,6 +24,19 @@ export const SingleArticle = () => {
             });
     }, [article_Id]);
 
+    useEffect(() => {
+        getComments(article_Id, {sortBy: 'created_at', order: 'desc'})
```

Comment:
the second argument being passed here is not being utilised or expected by the function - try to make sure all code added is relevant to the functionality you are trying to implement.